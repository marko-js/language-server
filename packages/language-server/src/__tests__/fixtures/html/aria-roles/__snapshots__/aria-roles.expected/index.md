## Diagnostics
### Ln 1, Col 6
```marko
> 1 | <div role="fake-role"></div>
    |      ^^^^ Type '"fake-role"' is not assignable to type 'AttrMissing | "none" | "search" | "list" | "link" | "dialog" | "grid" | "listbox" | "menu" | "tree" | "alert" | "alertdialog" | "application" | "article" | "banner" | ... 55 more ... | "treeitem"'.
  2 |
```

### Ln 1, Col 2
```marko
> 1 | <div role="fake-role"></div>
    |  ^^^ Fix all of the following:
  Role must be one of the valid ARIA roles: fake-role
  2 |
```

