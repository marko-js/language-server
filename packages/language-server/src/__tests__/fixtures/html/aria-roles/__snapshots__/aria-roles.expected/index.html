<div data-marko-node-id="0" role="fake-role"></div>